packages/react/src/lib/mergeProviderContexts.ts

```diff
@@ -1,7 +1,33 @@
-import felaRenderer from './felaRenderer'
 import { ProviderContextPrepared, ProviderContextInput } from '../types'
+import { Renderer } from '../themes/types'
+import { createRenderer, felaRenderer } from './felaRenderer'
 import mergeThemes from './mergeThemes'
 
+const registeredRenderers = new WeakMap<Document, Renderer>()
+
+export const mergeRenderers = (
```

Comment:
please add a uni test to document the function

Reply:
Done 👍